### ISSUE TITLE
rename nwp service to include metoffice

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/ocf-infrastructure#469]
Rename nwp service. To be nwp to nwp-metoffice https://github.com/openclimatefix/ocf-infrastructure/blob/main/terraform/nowcasting/development/main.tf#L138

Similarily replace `data-national` with `data-metoffice`

Would be good to do this on `development` first then we can move over to `production`

**Comment by aatmanvaidya** [comment]
Hello @peterdudfield, I would like to take this issue up

If I understand it correctly, I just have to rename the `nwp` to `nw-metoffice`, so will I have to change the `ecs-task_name` name variable right? Is there something else that will have to be done

Also, can you give me some more insights on how to setup `ocf-infrastructure` to test this out, after following this [setup guide](https://github.com/openclimatefix/ocf-infrastructure/blob/main/terraform/nowcasting/development/README.md), will I have to run some tests?

**Comment by peterdudfield** [comment]
Hi @aatmanvaidya thanks so much for taking this up.

Yea thats right, `nwp` to `nwp-metoffice`. 

testing is now done through CI. So unless you know terraform very well, it might be good to just let the CI do its thing. Sorry if that is very helpful. I would b interested to know if we can test it a different way. `terraform validate` will get you so far

**Comment by aatmanvaidya** [comment]
@peterdudfield understood, will work on this and make a PR at the earliest!

**Comment by peterdudfield** [comment]
Thank you

**Comment by aatmanvaidya** [comment]
@peterdudfield I have some small doubts regarding setting up `terraform`

When I did `terraform init` inside `nowcasting/development`, I got an error saying I will have to do `terraform login` first, I created a login token, but now where do I store/enter this login token? Will I have to create a [config file](https://developer.hashicorp.com/terraform/cli/commands/login)?

Apologies for a silly question, I am a beginner to terraform.
